### PR TITLE
Use either minAvailable or maxUnavailable for logs pdb

### PR DESCRIPTION
**Pull request by poblahblahblah**
This change updates the logs pdb.yaml to either use maxUnavailable, if defined, or minAvailable as a default.

The current behavior is to always include minAvailable since it's specified as a default in values.yaml file. There's no way in helm to unset this. With this change it will use maxUnavailable if defined and if not it will fall back to minAvailable.

### Checklist

- [x] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

**Comment by poblahblahblah** [comment]
Values:

```
metadata:
  logs:
    podDisruptionBudget:v
    maxUnavailable: 1
```

Before:

```
# Source: ttd-sumologic-kubernetes-collection/charts/sumologic/templates/logs/common/pdb.yaml
apiVersion: policy/v1beta1
kind: PodDisruptionBudget
metadata:
  name: release-name-sumologic-otelcol-logs-pdb
  namespace: ttd-system
  labels:
    chart: "sumologic-3.9.0"
    release: "release-name"
    heritage: "Helm"
spec:
  selector:
    matchLabels:
      app: release-name-sumologic-otelcol-logs
  maxUnavailable: 1
  minAvailable: 2
```

After:

```
# Source: ttd-sumologic-kubernetes-collection/charts/sumologic/templates/logs/common/pdb.yaml
apiVersion: policy/v1beta1
kind: PodDisruptionBudget
metadata:
  name: release-name-sumologic-otelcol-logs-pdb
  namespace: ttd-system
  labels:
    chart: "sumologic-3.9.0"
    release: "release-name"
    heritage: "Helm"
spec:
  selector:
    matchLabels:
      app: release-name-sumologic-otelcol-logs
  maxUnavailable: 1
```

**Comment by poblahblahblah** [comment]
I added in a CHANGELOG item but the check is still failing - not entirely sure why!

**Comment by sumo-drosiek** [comment]
@poblahblahblah Please use `make add-changelog-entry`

**Comment by poblahblahblah** [comment]
@sumo-drosiek done - thank you for the pointer!

File: .changelog/3231.fixed.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Use either minAvailable or maxUnavailable for logs pdb
```
Comment (sumo-drosiek) [inline .changelog/3231.fixed.txt:1]:
```suggestion
feat: use either minAvailable or maxUnavailable for logs pdb
```

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**